### PR TITLE
fix: resnet and add cd

### PR DESCRIPTION
**Pull request by cristianmtr**
- fix docstring about resnet and torchvision link
- add resnet18 as default
- add CD

### CODE REVIEW

File: torch_encoder.py [hunk 1]
```diff
@@ -23,13 +23,13 @@ class ImageTorchEncoder(Executor):
 
     Internally, :class:`ImageTorchEncoder` wraps the models from
     `torchvision.models`.
-    https://pytorch.org/docs/stable/torchvision/models.html
+    https://pytorch.org/vision/stable/models.html
 
     :param model_name: the name of the model. Supported models include
         ``alexnet``, `squeezenet1_0``,  ``vgg16``,
```
Comment (JoanFM) [inline torch_encoder.py:29]: I think the list is much larger, because all other resnets should be available as well

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: torch_encoder.py (hunk 1 above)
Comment (cristianmtr) [reply torch_encoder.py:29]: Then we can remove it and just reference the link

Comment (cristianmtr) [reply torch_encoder.py:29]: I'm now specifying that the list is not complete and that they should check the link.

**Comment by cristianmtr** [comment]
@JoanFM @slettner recheck

**Review by JoanFM — APPROVED** [review]

**Pull request merged**